backend/geolibre_server_api/tests/test_api.py

```
@@ -275,3 +574,1213 @@ def test_validation_and_errors_use_contract_shape(client):
+    group = client.post(
+        "/api/groups",
+        headers=auth(admin),
+        json={"name": "Editors", "sharedUpdate": True},
+    ).json()["group"]
+    client.put(
+        f"/api/groups/{group['id']}/members",
+        headers=auth(admin),
+        json={"username": "member", "role": "member"},
+    )
+
+    def create(token, title, visibility, organization_id=None, group_ids=None):
```

Ruff (0.16.1)

[warning] Missing return type annotation for private function `create`

(ANN202)